doc/doxygen/src/using-rust.md

```diff
@@ -131,3 +131,18 @@ $ cargo +nightly-2019-12-05 install --path c2rust
 [cargo]: https://doc.rust-lang.org/cargo/
 [**rustup**, installed as described on its website]: https://rustup.rs/
 [introduction post]: https://immunant.com/blog/2019/08/introduction-to-c2rust/
+
+Maintenance
+-----------
+
+The riot-sys and riot-wrappers crates are currently maintained outside of the RIOT project,
+primarily for practical historical reasons
+(all their CI has been set up on GitLab in the orgenization it was developed in).
```

Comment:
```suggestion
(all their CI has been set up on GitLab in the organization it was developed in).
```
Can't comment on the content but while reading through the newest PRs I noticed this typo. ;-)

Reply:
Thanks, fixed!